Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 Bugfixes:
  * Control Flow Graph: Fix incorrectly reported unreachable code.
  * Solc-Js: Use the same optimizer settings as ``solc`` when the ``--optimize`` flag is not specified.
+ * Standard JSON: Fix missing ``settings.optimizer.enabled`` key not being equivalent to setting it to ``false``.
```

Comment:
Actually can you write a bit more about what it was doing? I mean just that people don't think that the optimizer was fully enabled.

Reply:
Sure.

Reply:
Updated with the text I posted on the chat.